src/recipe.js

```diff
@@ -1,15 +1,25 @@
+let ingredientsDB = require('../src/data/ingredient-data');
+
 class Recipe {
   constructor(recipe) {
     this.id = recipe.id;
     this.name = recipe.name;
     this.image = recipe.image;
     this.tags = recipe.tags;
     this.ingredients = recipe.ingredients;
+    this.instructions = recipe.instructions;
   }
   calculateIngredientsCost() {
-    // return this.ingredients.map(i => {
-    //   ingredientData.find(ingredient => ingredient === i);
-    // });
+    let totalInCents = this.ingredients.reduce((total, item) => {
+      let itemPrice = ingredientsDB.default.find(ingredient => ingredient.id === item.id).estimatedCostInCents;
+      return total += itemPrice * item.quantity.amount;
+    },0);
+    let dollars = totalInCents / 100;
+
+    return dollars = dollars.toLocaleString("en-US", {style:"currency", currency:"USD"});
+  }
+  getInstructions() {
+    return this.instructions;
   }
```

Comment:
This method is redundant, if we want a method that gets the instructions, it should probably be parsing out the instructions into a more usable data type for DOM manipulation. For example, we could turn each object into a string and return an array of strings like this:
```
[
'1: In a large mixing bowl, whisk together the dry ingredients (flour, pudding mix, soda and salt). Set aside.In a large mixing bowl of a stand mixer, cream butter for 30 seconds. Gradually add granulated sugar and brown sugar and cream until light and fluffy.',
'2: Add egg and vanilla and mix until combined.',
'3: Add dry ingredients and mix on low just until incorporated. Stir in chocolate chips.Scoop the dough into 1,5 tablespoon size balls and place on a plate or sheet. Cover with saran wrap and chill at least 2 hours or overnight.When ready to bake, preheat oven to 350 degrees.',
'4: Place the cookie dough balls into ungreased muffin pan. Sprinkle with sea salt.',
'5: Bake for 9 to 10 minutes, or until you see the edges start to brown.',
'6: Remove the pan from the oven and let sit for 10 minutes before removing onto a cooling rack.Top with ice cream and a drizzle of chocolate sauce.'
]
```

Reply:
I figured we would want to just output the text into an `<ol></ol>` that would save us from having to worry about the numbers. So since we are returning an array of objects. we can just run a forEach loop to interpolate a `<li></li>` with the text. Then those list items would automatically get assigned to the correct numbered position.

Reply:
I thought displaying them in an ordered list would be a little nicer on the eyes. What do you think?

Reply:
That sounds good, but I don't think we need this method then, since we can get the array of objects already by accessing the ingredients property

Reply:
Oh yeah, lol. I'll remove this one.